en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-2-all-in-one-gw.md

```
@@ -30,10 +30,10 @@ This is the standard distributed deployment for API Manager. The default configu
         - [2.4 Configure JWKS URL](#24-configure-jwks-url)
         - [2.5 Deploy All-in-One](#25-deploy-all-in-one)
         - [2.6 Enable High Availability](#26-enable-high-availability)
-    - [3. Universal Gateway Configuration](#3-universal-gateway-configuration)
+    - [3. Classic Gateway Configuration](#3-universal-gateway-configuration)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)